assets/scripts/ui/__tests__/CloseButton.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jest */
+import React from 'react'
+import { shallow } from 'enzyme'
+import CloseButton from '../CloseButton'
+
+const onClick = jest.fn()
+
+describe('CloseButton', () => {
+  it.only('should renders without crashing', () => {
+    const wrapper = shallow(<CloseButton onClick={onClick} />)
+    expect(wrapper.find('button .close').length).toEqual(1)
```

Comment:
Is this the right selector? This is saying `.close` as a child of `button`, but it should be the same element, right?

We should also add more tests to make sure that the `className`, `title`, and `onClick` props are working, if provided.

Reply:
> Is this the right selector? This is saying .close as a child of button, but it should be the same element, right?

Yeah, It should be the same, should have been `button.close`